amida-tech/amida-auth-microservice
# Feature/add UUID

If this PR fixes a bug, you _must_ add test cases representative of the bug.
#### What's this PR do?
Adds table creation to the migration model.  Table migration by default checks if table exists, so verification and returning true is not needed. Migrations and table creation will happen using the command `yarn migrate` and migrations/table creation can be undone using `yarn migrate:undo`

#### Related JIRA tickets:
#### How should this be manually tested?

1.  run `yarn` to verify all packages are up to date
2. using a new empty database, run `yarn migrate` 
3. run `yarn start` 

No errors should occur. 

If you also have an existing database with tables, follow the same steps:
1.  run `yarn` to verify all packages are up to date
2. using an EXISTING database, run `yarn migrate` 
3. run `yarn start`

New migrations should automatically add migration files to sequelize_meta table, and the existing tables will not be impacted. 

#### Any background context you want to provide?
